### PR TITLE
Learn How It works banner

### PR DESCRIPTION
**Pull request by mitia-anah**
Hey Team

The 'How It works' page is done. You now can have a look at it. Comments are highly appreciated.

ClickUp: https://app.clickup.com/t/45e4n1
Refs: #45e4n1 - C30
Thanks

### CODE REVIEW

**Review by lightme-fan — APPROVED** [review]
Looking good! Great job.

**Review by Tiaraha-funny — APPROVED** [review]

File: src/components/IntroContent/IntroContent.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import { mediaQueries } from '../../mediaQueries/mediaQueries'
+import Button from '../Buttons/Buttons'
+
+export interface Props {
+  paragraph: string
+  title: string
+  imageUlr: string
+}
+
+export const IntroContent = ({ paragraph, title, imageUlr }) => {
```
Comment (andrzej-onja) [inline src/components/IntroContent/IntroContent.tsx:12]: I dont think we need those props, just hardcode the text in component

File: src/App.tsx [hunk 2]
```diff
@@ -1,21 +1,7 @@
...
-  const ads = useSelector(adsSelector)
-
-  useEffect(() => {
-    dispatch(fetchAds([]))
-  }, [dispatch])
-
   return (
     <div className='App'>
-      <OffersList offers={ads} />
+      <p>IntroContent</p>
```
Comment (andrzej-onja) [inline src/App.tsx:4]: I think it should be ;
```
<IntroContent />
<OffersList offers={ads} />

Comment (andrzej-onja) [reply src/App.tsx:4]: keep the offers logic and fetching as it was

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**